src/src/components/navigation.js

```diff
@@ -1,27 +0,0 @@
-import React, { useCallback, useContext } from 'react';
-import { useLocation } from 'react-router-dom';
-import { FalconApiContext } from '../contexts/falcon-api-context.js';
-import { SlTab, SlTabGroup, SlTabPanel } from '@shoelace-style/shoelace/dist/react';
-
-function TabNavigation({ children }) {
-  const location = useLocation();
-  const { navigation} = useContext(FalconApiContext);
-
-  const doNavigate = useCallback((name) => {
-    switch(name) {
-      case 'home': return navigation.navigateTo({ path: '/' });
```

Comment:
We need this specific navigation to be called, without that app won't navigate inside iframe. Does it work for you without it?

Reply:
Thanks for the context around this.  I included a Link component to help with navigation inside and outside of an extension/page https://github.com/CrowdStrike/foundry-js-blueprint-react/pull/1/files#diff-d69779a774d18aab6696dcf281454982a329104177c1a5f885089da9fefefd94